common/src/main/java/io/druid/metadata/MetadataStorageConnectorConfig.java

```diff
@@ -38,8 +38,8 @@
   @JsonProperty
   private String user = null;
 
-  @JsonProperty
-  private String password = null;
+  @JsonProperty("password")
```

Comment:
may be we can add it to the list of [extendable] things at http://druid.io/docs/latest/Modules.html